guide/inheritance-wallet/wallet-creation.md

```diff
@@ -0,0 +1,289 @@
+
+### Wallet configuration
+
+The wallet creation flow consists of two main parts. The first is the wallet configuration, where Bob and Alice define the rules for how the wallet should work. In the second part, they add all the signing keys.
+
+#### Primary key set
+
+Bob starts the wallet creation process on his phone, while Alice will use the app on her own phone to add her signing key. The app gives them a choice between two of the most commonly used wallet types: a standard, single-key wallet or a 2-of-3 multi-key wallet, which is suited for larger savings. 
+
+{% include image-gallery.html pages = page.images_wallet-configuration %}
+
+There is also a third option that allows users to create their own custom setups. But Bob and Alice choose the multi-key approach.
+
+#### Recovery path
+
+After configuring the primary key set, the app asks the users whether they would like to enable a recovery path. We have covered this in the time-based reference design as well. Here we show an alternative design option for this same feature.
+
+{% include image-gallery.html pages = page.images_recovery-path %}
+
+We could also call the recovery path a recovery key set, since it technically is just that. However, the point of the recovery path is that we want to use the same keys that are used in the primary key set. To make this distinction more clear, we treat recovery paths as a different feature from alternative key sets in our application. You might want to take a different approach in your own application, based on your specific user base.
+
+#### Inheritance key set
+
+After enabling the recovey path, the app offers users to create a dedicated inheritance key set. This works in more or less the same way as for the primary key set.
+
+The main exception is that the inheritance key set should only be unlocked after a certain amount of time. After choosing the appropriate key set type, Bob is prompted to define the rules under which the key set should be unlocked.
+
+{% include image-gallery.html pages = page.images_inheritance-key-set %}
+
+## Adding the signing keys
+
+The wallet is now fully configured. What is left to do is to add the necessary amount signing keys.
+
+#### Primary keys
+
+Adding a key technically means that users have to import the extended public key (XPUB) from each signing device that should be used to sign transactions. 
+
+This sounds more complicated than it actually is. Depending on the specific signing devices that are being used, it can be as simple as scanning a QR code, connecting it via NFC or Bluetooth, or by using a USB cable. Bob does just that and imports the first two keys by himself.
+
+{% include image-gallery.html pages = page.images_primary-keys-bob %}
+
+Our application allows the users to create the wallet collaboratively. This means that each key holder can use their own phone and signing device to add their key. So Bob creates a key request that Alice can use.
```

Comment:
That's a very good point. Thanks for bringing it up!

My original idea was that the app is optimized for collaborative wallet creation. The idea is that "requesting" a key is really just sharing the QR-encoded wallet details. Each request contains a unique identifier that prevents the same invite from being used twice and also defines the key set to which the XPUB will belong (primary or inheritance). 

During that process, the app (wallet provider) would keep track of this and keep adding the XPUBs to the assigned key set as they come in. You're right that this implies the wallet provider of storing the configuration somewhere, until wallet creation is finished. Since there is no signup required, the wallet provider would only see the configuration of the wallet with no additional data. But, as you point out, it introduces risk for the users, as the provider can tamper with the wallet config. 

Might well be that I'm over-emphasizing the asynchronous and remote use case. After all, meeting physically (or remotely over video call) is probably a reasonable thing to do in a family.